client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/TableFilter.tsx

```diff
@@ -93,6 +94,40 @@ const TableFilter = (props: Props) => {
                     )}
                     limitTags={1}
                 />
+                <Typography className={classes.title} >
```

Comment:
fix layout of Table filter. On small screen can't see all fields
![image](https://user-images.githubusercontent.com/55889135/105017856-f33fef80-5a4c-11eb-9f13-efc11056cd85.png)